src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -22,10 +22,34 @@ function jcms_notifications_drush_command() {
       'drush article-import' => 'Import articles from Lax and return a message when finished.',
     ),
   );
+  $items['article-import-all'] = array(
+    'description' => 'Imports all articles from Lax.',
+    'drupal dependencies' => array('jcms_notifications'),
+    'aliases' => array('aia'),
+    'examples' => array(
+      'drush article-import-all' => 'Import articles from Lax and return a message when finished.',
+    ),
+  );
   return $items;
 }
 
 /**
+ * Callback function drush_jcms_notifications_article_import_all().
+ */
+function drush_jcms_notifications_article_import_all() {
+  $fetch_service = \Drupal::service('jcms_notifications.fetch_article_service');
+  $crud_service = \Drupal::service('jcms_notifications.article_crud_service');
+  $ids = $fetch_service->getAllArticleIDs();
+  if ($ids) {
+    foreach ($ids as $id) {
+      $article = $fetch_service->getArticle($id);
+      $crud_service->crudArticle($article);
+      drush_print(dt('Processed article !article_id', ['!article_id' => $id]));
```

Comment:
Yes there is a process (`drush ai`), the problem we encountered in search was that the memory of the bulk import grows and grows. However, it may be the SDK that was filling it, while this project isn't using it (I think?). Testing this assumption: can you run this locally pointing to `prod--gateway.elifesciences.org/articles` and successfully import 3000+ articles without going out of memory?